cosmo_tester/framework/assertions.py

```diff
@@ -0,0 +1,39 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+
+from influxdb import InfluxDBClient
+
+
+def assert_general_deployment_data(self, influx_client):
```

Comment:
is this method supposed to be called from outside this module as well? if not, it should have an underscore prefix

note that if this method is a public method (i.e. no underscore prefix), its name is rather ambiguous; the name made (some) sense before, when it was a private method, since it was called from a "assert_monitoring_.." method. If you check the nodecellar_test code, you can see that  the assert_monitoring_data_exists calls both this and another method which verifies mongodb collector data - which is the reason for the original, still a-bit ambiguous name.